### PR TITLE
(feat) webui: Add some minor UI changes

### PR DESCRIPTION
**Pull request by Kussie**
## Description
I was bored and added a few tweaks to the UI that i personally find useful.  But up to you to decide to add :)

* Adds library tag to Series and Book view, (Useful if navigating from a carousel on the Home screen to see what library the book/series belong too)
* Adds a button to Server Management to Empty Trash of all libraries at once.

## Translations Added/Required
* common.library = "Library"
* server.server_management.button_empty_trash = "Empty All Trash"

## Screenshots
Screenshot 2021-08-24 11 06 19 (2)
Screenshot 2021-08-24 11 05 50

### CODE REVIEW

**Comment by gotson** [comment]
You can add the English strings in `en.json`. Only the other languages are managed in Weblate.

**Comment by Kussie** [comment]
> You can add the English strings in `en.json`. Only the other languages are managed in Weblate.

Oh yes i forgot about that.  Added :)

File: komga-webui/src/locales/en.json [hunk 1]
```diff
@@ -628,6 +629,7 @@
   },
   "server": {
     "server_management": {
+      "button_empty_trash": "Empty All Trash",
```
Comment (gotson) [inline komga-webui/src/locales/en.json:632]:
```suggestion
      "button_empty_trash": "Empty trash for all libraries",
```

File: komga-webui/src/views/BrowseBook.vue [hunk 2]
```diff
@@ -233,6 +233,23 @@
         </v-row>
       </template>
 
+      <!--  Library    -->
+      <v-row class="align-center text-caption">
+        <v-col cols="4" sm="3" md="2" xl="1" class="py-1 text-uppercase">{{ $t('common.library') }}</v-col>
+        <v-col cols="8" sm="9" md="10" xl="11" class="py-1">
+          <v-chip
+            :class="$vuetify.rtl ? 'ml-2' : 'mr-2'"
+            :title="getLibraryName(series)"
```
Comment (gotson) [inline komga-webui/src/views/BrowseBook.vue:242]: shouldn't that be `getLibraryName(book)` ?

File: komga-webui/src/views/BrowseBook.vue [hunk 3]
```diff
@@ -233,6 +233,23 @@
...
+            :to="{name:'browse-libraries', params: {libraryId: series.libraryId }}"
+            label
+            small
+            outlined
+            link
+          >{{ getLibraryName(series) }}
```
Comment (gotson) [inline komga-webui/src/views/BrowseBook.vue:248]: here too

File: komga-webui/src/views/BrowseBook.vue (hunk 2 above)
Comment (Kussie) [reply komga-webui/src/views/BrowseBook.vue]: Yep fixed

File: komga-webui/src/views/BrowseBook.vue (hunk 3 above)
Comment (Kussie) [reply komga-webui/src/views/BrowseBook.vue]: Yep fixed

**Comment by gotson** [comment]
For the library name display, how about:

Screenshot 2021-08-24 at 10 30 33

or

Screenshot 2021-08-24 at 10 32 00

**Comment by Kussie** [comment]
> For the library name display, how about:
> 
> Screenshot 2021-08-24 at 10 30 33
> 
> or
> 
> Screenshot 2021-08-24 at 10 32 00

I like the first actually.  Keep it in line with the wording used in search results as well.  Want me to look at changing that around?

**Comment by gotson** [comment]
I liked the second better 🙃 , but it's true the first one is more in line with the search.

I'll send you a patch, i already did the code !

**Comment by gotson** [comment]
How about in the toolbar on the Series view?

Screenshot 2021-08-24 at 11 10 55

**Comment by Kussie** [comment]
> How about in the toolbar on the Series view?
> 
> Screenshot 2021-08-24 at 11 10 55

Looks good.  I didnt think of that, i initially start with breadcrumb style in the toolbar such as "Library / Series" but it wasnt great on mobile.  But just having the library below the series in the toolbar like that, solve it as well.

**Comment by gotson** [comment]
> > How about in the toolbar on the Series view?
> > Screenshot 2021-08-24 at 11 10 55
> 
> Looks good. I didnt think of that, i initially start with breadcrumb style in the toolbar such as "Library / Series" but it wasnt great on mobile. But just having the library below the series in the toolbar like that, solve it as well.

actually it kinda breaks the flow, and we can't have the same for the books, given the toolbar doesn't show the series.

I think at some point we will need to review all the screens, so far we just added stuff here and there, but in the end it's not always coherent between the different screens.

**Comment by Kussie** [comment]
> > > How about in the toolbar on the Series view?
> > > Screenshot 2021-08-24 at 11 10 55
> > 
> > 
> > Looks good. I didnt think of that, i initially start with breadcrumb style in the toolbar such as "Library / Series" but it wasnt great on mobile. But just having the library below the series in the toolbar like that, solve it as well.
> 
> actually it kinda breaks the flow, and we can't have the same for the books, given the toolbar doesn't show the series.
> 
> I think at some point we will need to review all the screens, so far we just added stuff here and there, but in the end it's not always coherent between the different screens.

Thats true, might be easier/better to go with the first solution in your initial screenshots (Under the title not in the toolbar) for consistency sake for the moment, until some of these areas get a bit of an overhaul at some point down the line

**Comment by gotson** [comment]
I decided on displaying next (second screenshot) on `smAndUp`, and below on `xsOnly`, just pushed some commits

**Comment by Kussie** [comment]
👍

**Pull request merged**